Yoast/yoast-components
# Add yoast warning component

## Summary

This PR can be summarized in the following changelog entry:

* Adds an exclamation triangle to the SvgIcon component.
* Adds a YoastWarning component that consists of an exclamation triangle followed by a message on a yellow background.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Fetch remote branches: `git fetch --all`.
* Switch to this branch: `git checkout 11700-add-yoast-warning-component`.
* Install dependencies: `yarn`.
* Run the stand-alone: `yarn start`.
* Go to this url in your browser: `http://localhost:3333/`.
* Go to the `Components` page (top right).
* Verify that the Yoast warning is there and looks like this:
<img width="1094" alt="yoastwarning" src="https://user-images.githubusercontent.com/35524806/49376139-9404b100-f706-11e8-9b79-88c17404b7ee.png">

* Switch the language direction.
* Verify that the Yoast warning looks like the following. Watch the margins between the icon and the message.
<img width="986" alt="yoastwarning_rtl" src="https://user-images.githubusercontent.com/35524806/49376272-05446400-f707-11e8-9d8b-a96b088a2327.png">


Fixes https://github.com/Yoast/wordpress-seo/issues/11700
